internal/tfengine/config.go

```diff
@@ -135,3 +127,39 @@ func loadConfig(path string, data map[string]interface{}) (*Config, error) {
 	}
 	return c, nil
 }
+
+func hclToJSON(b []byte) ([]byte, error) {
+	// Directly trying to unmarshal to cty.Value doesn't seem to work,
+	// so wrap in a dummy field.
+	wrap := struct {
```

Comment:
Lower down, you use the construct `var jr struct { ... }`, while here you do what (I think) is the same thing but with `wrap := struct { ... }{}`.

Should choose one and use it for consistency. Personally I slightly prefer `var` over `:=`.